tobac/tracking.py

```diff
@@ -113,12 +113,18 @@ def linking_trackpy(
         )
     elif method_linking == "predict":
 
+        # avoid setting pos_columns by renaimng to default values to avoid trackpy bug
+        features.rename(columns={"hdim_1": "x", "hdim_2": "y"}, inplace=True)
```

Comment:
I'm not sure that it matters as long as it's consistent, but I believe that elsewhere in the code, `hdim_1` is typically the `y` dimension. But I suppose that it would be the other if users input an array that had dimensions `nx, ny`? I'm happy with this as-is.

Reply:
I recommend swapping as Sean mentioned for the sake of consistency.

Reply:
Ok, let's keep this consistent then.